src/mixins/raster-layer-point-mixin.js

```diff
@@ -527,6 +541,17 @@ export default function rasterLayerPointMixin(_layer) {
         }
       }
     } else {
+      parser.writeSQL({
```

Comment:
Leftover stuff? Or does this have some side effects?

Reply:
Good catch - that was some leftover stuff from experimenting